format/Flight.proto

```diff
@@ -426,8 +426,43 @@ message Ticket {
 }
 
 /*
- * A location where a Flight service will accept retrieval of a particular
- * stream given a ticket.
+ * A location to retrieve a particular stream from. This URI should be one of
+ * the following:
+ *  - An empty string or the string 'arrow-flight-reuse-connection://?':
+ *    indicating that the ticket can be redeemed on the service where the
+ *    ticket was generated via a DoGet request.
+ *  - A valid grpc URI (grpc://, grpc+tls://, grpc+unix://, etc.):
+ *    indicating that the ticket can be redeemed on the service at the given
+ *    URI via a DoGet request.
+ *  - A valid HTTP URI (http://, https://, etc.):
+ *    indicating that the client should perform a GET request against the
+ *    given URI to retrieve the stream. The ticket should have been empty
+ *    in this case and should be ignored by the client.
+ *  - An object storage URI (s3://, gs://, abfs://, etc.):
+ *    indicating that the client should retrieve the data from the provided
+ *    object storage location. The ticket should be empty in this case and
+ *    should be ignored by the client.
+ *
+ * We allow non-Flight URIs for the purpose of allowing Flight services to indicate that
+ * results can be downloaded in formats other than Arrow (such as Parquet) or to allow
+ * direct fetching of results from a URI to reduce excess copying and data movement.
+ * In these cases, the following conventions should be followed by servers and clients:
+ *
+ *  - Unless otherwise specified by the 'Content-Type' header of the response,
+ *    a client should assume the response is an Arrow IPC Stream. Usage of an IANA
```

Comment:
```suggestion
 *    a client should assume the response is an Arrow IPC Streaming format. Usage of an IANA
```

Reply:
semantically this doesn't make much sense. I'll reword it a bit to use `Streaming format` if that's preferred